tests/decryption/test_decrypt.py

```diff
@@ -97,3 +101,47 @@ def file_contents_are_valid():
         assert files_exist()
 
         assert file_contents_are_valid()
+
+
+class TestMoveFiles:
+    """Test move_files."""
+
+    @pytest.fixture()
+    def files(self, tmp_path):
```

Comment:
Maybe better to give a more descriptive name, perhaps `get_files()`?

Reply:
Would something like `input_files` or `file_copies` be better? Since it's a fixture I reference it like a variable rather than a function.

Reply:
Fair enough. In that case, I guess you can just leave it.